plugins/cue_points/quiz/lib/api/enums/KalturaQuizOutputType.php

```diff
@@ -0,0 +1,9 @@
+<?php

```

Comment:
I see that you created new file for KalturaQuizOutputType but I don't see that you deleted old KalturaQuizFileType.